Enforcement.md

```diff
@@ -41,18 +43,16 @@ Possible responses may include:
 *   A private reprimand from the committee to the individual(s) involved. In this case, the committee chair will deliver that reprimand to the individual(s) over email, cc'ing the committee.
 *   A public reprimand. In this case, the committee chair will deliver that reprimand in the same venue that the violation occurred (e.g. in IRC for an IRC violation; email for an email violation). The committee may choose to publish this message elsewhere for posterity.
 *   An imposed vacation (e.g. asking someone to "take a week off" from a mailing list or IRC). The committee chair will communicate this "vacation" to the individual(s). They'll be asked to take this vacation voluntarily, but if they don't agree then a temporary ban may be imposed to enforce this vacation.
-*   A permanent or temporary ban from some or all JS Foundation spaces (including mailing lists and IRC). The committee will maintain records of all such bans so that they may be reviewed in the future, extended to new JS Foundation fora, or otherwise maintained.
+*   A permanent or temporary ban from some or all TC39 spaces. The committee will maintain records of all such bans so that they may be reviewed in the future, extended to new TC39 fora, or otherwise maintained.
 *   A request for a public or private apology. The chair will deliver this request. The committee may, if it chooses, attach "strings" to this request: for example, the committee may ask a violator to apologize in order to retain his or her membership on a mailing list.
 
 Once a resolution is agreed upon, but before it is enacted, the committee will contact the original reporter and any other affected parties and explain the proposed resolution. The committee will ask if this resolution is acceptable, and must note feedback for the record. However, the committee is not required to act on this feedback.
 
-Finally the committee will make a report for the JS Foundation Board (as well as the project teams in the event of an ongoing resolution, such as a ban).
-
-The committee will never publicly discuss the issue; all public statements will be made by the JS Foundation Board.
+Finally the committee will make a report for the TC39 ???. The committee will never publicly discuss the issue; all public statements will be made by the TC39 ???.
```

Comment:
> The committee will never publicly discuss the issue

I like this being in a new paragraph and we can even bold this part, it addresses some of the already expressed concerns.

> ... make a report for the TC39 ???

make a report of the TC39 members and respective delegates and the ECMA secretariat (?)

> ... public statements will be made by the TC39 ???

TC39 Chair and/or the ECMA Secretariat?